python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -26,6 +25,7 @@
 from Ganga.GPIDev.Schema import ComponentItem, FileItem, GangaFileItem, Schema, SimpleItem, Version
 from Ganga.Runtime.spyware import ganga_job_submitted
 from Ganga.Utility.Config import ConfigError, getConfig
+from Ganga.Utility.external.OrderedDict import OrderedDict as oDict
```

Comment:
If this import was simply:
```python
from Ganga.Utility.external.OrderedDict import OrderedDict
```
it would simplify the porting process when we can depend on Python 2.7.